metricbeat/module/rabbitmq/exchange/exchange_test.go

```diff
@@ -61,16 +65,27 @@ func TestFetchEventContents(t *testing.T) {
 }
 
 func TestData(t *testing.T) {
-	server := mtest.Server(t, mtest.DefaultServerConfig)
-	defer server.Close()
+	t.Skip("Skipping `data.json` generation test")
```

Comment:
There is a check in the `WriteEventsReporterV2` for the `-data` flag. If we want to skip it already here, we should do it based on the `-data` flag instead of always skipping it.